setup.py

```diff
@@ -15,22 +16,35 @@
 
 PATH_ROOT = os.path.dirname(__file__)
 builtins.__LIGHTNING_BOLT_SETUP__ = True
+builtins.__LIGHTNING_SETUP__ = True
+
+import sys
+
+# remove local path to force loading global models
+removed_locals = False
+if '' in sys.path:
+    sys.path.remove('')
+    removed_locals = True
+import pytorch_lightning
```

Comment:
```suggestion
try:
    import pytorch_lightning
except ImportError:
    print('you are missing `pytorch_lightning`, you need to install it first.')
```

Reply:
this is silly... lightning should just be a hard dep

Reply:
it is, but currently you first need to install lightning and then bolts and you can't rely on auto-dependency installs since this is not (yet) working. See my comment above

Reply:
got it ok

Reply:
I believe that there shall be a way to inject the init in setup time, some wrapper or calling a function inbeween start setup and real instalation

Reply:
I will check again this https://docs.python.org/3/library/pkgutil.html